### PR TITLE
Albrja/mic 5240/slack developer

### PR DESCRIPTION
**Pull request by albrja**
### Slack developer when Jenkins pipeline fails
- *Category*: Feature
- *JIRA issue*: [MIC-5240](https://jira.ihme.washington.edu/browse/MIC-5240)
- *Research reference*: 

### Changes and notes
-slack developer if build fails

### CODE REVIEW

**Review by patricktnast — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -1,3 +1,15 @@
+def githubUsernameToSlackName(github_author) {
+  // Add team members as necessary
+  def mapping = [
```
Comment (rmudambi) [inline Jenkinsfile:3]: This is fine for now, but in the future, I think we could move this onto the cluster file-sytem

File: Jenkinsfile [hunk 2]
```diff
@@ -1,3 +1,15 @@
+def githubUsernameToSlackName(github_author) {
```
Comment (rmudambi) [inline Jenkinsfile:1]: Is this mapping correct for our stash repos?

**Review by rmudambi — APPROVED** [review]
Wow 57 commits!

File: Jenkinsfile (hunk 2 above)
Comment (albrja) [reply Jenkinsfile:1]: Ill test that out

**Pull request merged**